weldx/asdf/file.py

```diff
@@ -525,3 +544,87 @@ def _show_interactive(
     @staticmethod
     def _show_non_interactive(buff: BytesIO):
         print(get_yaml_header(buff))
+
+
+# Methods of UserDict do not follow the NumPy doc convention.
```

Comment:
Can sphinx not work with their default docstring format? (rst?)

Reply:
The docstrings looked ugly, because of the different convention. E.g. the summary line is empty